data_import/serializers.py

```diff
@@ -42,12 +42,29 @@ def to_representation(self, instance):
         ret["id"] = instance.id
         ret["basename"] = instance.basename
         ret["created"] = instance.created
+        ret["datatypes"] = self.get_file_datatypes(instance)
         ret["download_url"] = instance.download_url(request)
         ret["metadata"] = instance.metadata
         ret["source"] = instance.source
+        ret["source_project"] = self.get_source_project(instance)
 
         return ret
 
+    def get_file_datatypes(self, obj):
+        """
+        Get links to DataType API endpoints for file DataTypes
+        """
+        return [
```

Comment:
As @beaugunderson mentioned, it would be good to give the full URL including the `BASE_URL` here to make using those endpoints easier. Also helps making it more consistent as the `download_url` return is already giving the full URL as well.